src/Serilog/Core/LoggingLevelSwitch.cs

```diff
@@ -31,6 +32,11 @@ public LoggingLevelSwitch(LogEventLevel initialMinimumLevel = LogEventLevel.Info
         _minimumLevel = initialMinimumLevel;
     }
 
+    /// <summary>
+    /// The event arises when <see cref="MinimumLevel"/> changed.
```

Comment:
Needs a comment pointing out that the event is raised under a lock (which will lead to deadlock if a handler waits on another lock `B`, which is already held by a thread that's attempting to change the level and is blocked on our lock here).

Reply:
done